app/Http/Controllers/BookController.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Book;
+use App\Http\Requests\BookCreateRequest;
+use Illuminate\Auth\Events\Validated;
+use Illuminate\Http\Request;
+
+class BookController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth');
```

Comment:
this is not a bug but the cleaner and simplest way to assign general middlewares like `auth` is doing it in the route files

Reply:
I guess you are right.. it is easier to control and implement when it is in route file..